app/views/nav/toggle_collapse_sidebar.js.erb

```diff
@@ -0,0 +1,9 @@
+$('.collapse-nav').find('input[type="submit"]').prop('disabled', false).removeClass("disabled")
```

Comment:
Actually, all view templates are implemented in HAML. Can you convert this to HAML?